ui-admin/src/study/participants/datarepo/DatasetList.tsx

```diff
@@ -0,0 +1,145 @@
+  const { user } = useUser()
+
+  const datasetColumns = useMemo<ColumnDef<DatasetDetails, string>[]>(() => {
+    return [{
+      id: 'select'
+    }, {
+      id: 'datasetName',
+      header: 'Dataset Name',
+      accessorKey: 'datasetName',
+      cell: info => <Link to={getDatasetDashboardPath(info.getValue() as unknown as string, currentEnvPath)}
+        className="mx-2">{info.getValue() as unknown as string}</Link>
+    }, {
+      id: 'datasetUuid',
+      header: 'Dataset ID',
+      accessorKey: 'datasetId'
+    }, {
+      id: 'created',
+      header: 'Created Date',
+      accessorKey: 'createdAt',
+      cell: info => instantToDefaultString(info.getValue() as unknown as number)
+    }, {
+      id: 'lastUpdated',
+      header: 'Last Export',
+      accessorKey: 'lastExported',
+      cell: info => instantToDefaultString(info.getValue() as unknown as number)
+    }, {
+      header: 'Terra Data Repo',
+      accessorKey: 'datasetId',
+      cell: info => <a href={
+        `https://jade.datarepo-dev.broadinstitute.org/datasets/${info.getValue()}`} target="_blank"
+      >View in Terra Data Repo <FontAwesomeIcon icon={faExternalLink}/></a>
+    }]
+  }, [datasets])
+
+  const datasetTable = useReactTable({
+    data: datasets,
+    columns: datasetColumns,
+    state: {
+      sorting: datasetsSorting
+    },
+    onSortingChange: setDatasetsSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel()
+  })
+
+  const contentHeaderStyle = {
+    padding: '1em 0 0 1em',
+    borderBottom: '1px solid #f6f6f6'
+  }
+
+  const loadData = async () => {
+    try {
+      //Fetch datasets
+      const datasets = await Api.listDatasetsForStudyEnvironment(
+        studyEnvContext.portal.shortcode,
+        studyEnvContext.study.shortcode,
+        studyEnvContext.currentEnv.environmentName)
+      const datasetsResponse = await datasets.json()
+      setDatasets(datasetsResponse)
+      setIsLoading(false)
+    } catch (e) {
+      Store.addNotification(failureNotification(`Error loading datasets`))
+    }
+  }
+
+  useEffect(() => {
+    loadData()
+  }, [])
+  return <div className="container-fluid py-3">
+    <h1 className="h3">Study Environment Datasets</h1>
+    { user.superuser &&
+        <button className="btn btn-secondary" onClick={() => setShowCreateDatasetModal(!showCreateDatasetModal)}
+          aria-label="show or hide export modal">
+          <FontAwesomeIcon icon={faPlus}/> Create new dataset
+        </button>
+    }
+    <CreateDatasetModal studyEnvContext={studyEnvContext}
+      show={showCreateDatasetModal}
+      setShow={setShowCreateDatasetModal}/>
+    <LoadingSpinner isLoading={isLoading}>
+      <div className="col-12 p-3">
+        <ul className="list-unstyled">
+          <li className="bg-white my-3">
+            <div style={contentHeaderStyle}>
+              <h6>Datasets</h6>
+            </div>
+            <table className="table table-striped">
```

Comment:
same, can be replaced with a call to `basicTableLayout`

Reply:
Done in 64eb80a3